### DISCUSSION TITLE
getting error for tabbed output

### DISCUSSION THREAD

**Discussion by ofajardo** [executablebooks/meta#141] [Q&A] [answered]
I'm trying to reproduce this example in the documentation:

https://jupyterbook.org/content/content-blocks.html#tabbed-content

in particular, I added a markdown cell and put this code in: (backslashes are there only to escape here for github markdown, I don't have them in the code).

```
\```{tabbed} Tab 1 title
My first tab
\```

\```{tabbed} Tab 2 title
My second tab with `some code`!
\```
```

However, when I build the book I get this warning:

```
50002: WARNING: Unknown directive type "tabbed".
```

and the markdown cell does not show at all in the rendered book.

my config file looks like this right now:

```
# Book settings
title: My sample book
author: The Jupyter Book Community
logo: logo.png

latex:
  latex_documents:
    targetname: book.tex

# Parse and render settings
parse:
  myst_extended_syntax      : true  # enable MyST extended syntax support (see documents for details)
  myst_url_schemes          : [mailto, http, https]  # URI schemes that will be recognised as external URLs in Markdown links
  
sphinx:
  extra_extensions:
    - sphinx_panels
```

I am using jupyter-book 0.8.1 with python 3.8.5 on Ubuntu 18.04

What could be wrong? It works OK with the dropdown example.

Thanks in advance!

**Comment by ofajardo** [comment]
Oh maybe I have to update some dependencies like myst parser or so? I'll check that tomorrow

**Reply by choldgraf** [reply]
yep if you upgrade jupyter book it should work now 👍

**Comment by ofajardo** [comment, marked as answer]
ok actually the problem is this: I upgraded to the last version of jupyter-book (0.8.1) and the issue was there.  The version of sphinx-panels that was installed was 0.4.1.

Now I upgraded sphinx-panels to 0.5 not without getting this warning:

```
jupyter-book 0.8.1 requires sphinx-panels~=0.4.1, but you'll have sphinx-panels 0.5.0 which is incompatible.
Successfully installed sphinx-panels-0.5.0
```

Now it works. However that means that the requirement in jupyter-book 0.8.1 is incorrect. I see that that is fixed in the master branch, so I guess it will be good on the next release.

**Reply by chrisjsewell** [reply]
0.8.2 was released a couple of hours ago 😉

**Reply by ofajardo** [reply]
Oh! Probably I was the last guy instaling 0.8.1!  Lol
